experimental/terraform/setup.sh

```diff
@@ -35,7 +35,7 @@ PARENT_PROJECT=$(gcloud config get-value project 2>/dev/null)
 if [[ ${AUTO} == 'true' ]]
 then
     # populate setup variables (for use in testing/dev)
-    make_new_project="y"
+    make_new_project='y'
```

Comment:
Why this change?  File seems to have a mix of double and single quotes.

Reply:
...and, um... it still has a mix! I was trying to find an error and noticed that this variable was set with double quotes, but then compared later to a literal with single quotes. I didn't think that would make a difference, but you know. So I made it consistent. Well, this one's consistent. The overall file isn't! 

It may be useful to apply a bash style guide... do you recommend one?